### PR TITLE
Customizing Wherehows to be Webster!

### PR DESCRIPTION
**Pull request by alisaraa**
Changing the logos
Changing the name of Wherehows on the login screen and tab
Editting readme

### CODE REVIEW

File: wherehows-web/app/templates/login.hbs [hunk 1]
```diff
@@ -1,16 +1,16 @@
...
+      Search database structure and definitions<br>Originally from LinkedIn&reg;
     </p>
     {{login-form username=username
                  password=password
                  errorMessage=errorMessage
                  onSubmit=(action "authenticateUser")}}
   </section>
 </section>
-{{outlet}}
+{{outlet}}
```
Comment (brafkind) [inline wherehows-web/app/templates/login.hbs]: was this intentional?

Comment (alisaraa) [reply wherehows-web/app/templates/login.hbs]: no...

File: README.md [hunk 2]
```diff
@@ -29,38 +29,24 @@ How WhereHows is used inside of LinkedIn and other potential [use cases][USE].
...
-
-Execute the [DDL files][DDL] to create the required repository tables in **wherehows** database.
+Please download the Amazon RedshiftJDBCDriver42.jar(https://docs.aws.amazon.com/redshift/latest/mgmt/configure-jdbc-connection.html#download-jdbc-driver) file
+Please download the gsp.jar for SQL Parsing(http://www.sqlparser.com/)
+Put a these jar files to **wherehows-etl/extralibs** directory. See [the download instrucitons][EXJAR] for more detail. ```cd WhereHows/wherehows-etl/extralibs```
+Please download Docker(https://www.docker.com/community-edition#/download)
 
 ### Build
-1. Get the source code: ```git clone https://github.com/linkedin/WhereHows.git```
-2. Put a few 3rd-party jar files to **wherehows-etl/extralibs** directory. Some of these jar files may not be available in Maven Central or Artifactory. See [the download instrucitons][EXJAR] for more detail. ```cd WhereHows/wherehows-etl/extralibs```
```
Comment (jaeger401) [inline README.md]: Sorry if I missed an earlier response to this, but was there a reason to leave this out? `gsp.jar` seems to be 100% required.

Comment (alisaraa) [reply README.md]: See line 38!

Comment (alisaraa) [reply README.md]: I thought it was better in the "download" than the "build" section.

**Pull request merged**